chromium_src/chrome/installer/setup/install_worker_unittest.cc

```diff
@@ -0,0 +1,180 @@
+      : example_version_(base::Version("1.0.0.0")),
+        example_path_(FILE_PATH_LITERAL("elevation_service.exe")) {}
+
+ protected:
+  base::Version example_version_;
+  base::FilePath example_path_;
+};
+
+// TODO(bsclifton): move this to a shared file. This was copied from:
+// chromium_src/chrome/install_static/brave_user_data_dir_win_unittest.cc
+class ScopedNTRegistryTestingOverride {
+ public:
+  ScopedNTRegistryTestingOverride(nt::ROOT_KEY root, const std::wstring& path)
+      : root_(root) {
+    EXPECT_TRUE(nt::SetTestingOverride(root_, path));
+  }
+  ~ScopedNTRegistryTestingOverride() {
+    nt::SetTestingOverride(root_, std::wstring());
+  }
+
+ private:
+  nt::ROOT_KEY root_;
+};
+
+// based on chrome/installer/setup/install_worker_unittest.cc
+class MockWorkItemList : public WorkItemList {
+ public:
+  MockWorkItemList() {}
+
+  MOCK_METHOD5(AddCopyTreeWorkItem,
+               WorkItem*(const base::FilePath&,
+                         const base::FilePath&,
+                         const base::FilePath&,
+                         CopyOverWriteOption,
+                         const base::FilePath&));
+  MOCK_METHOD1(AddCreateDirWorkItem, WorkItem*(const base::FilePath&));
+  MOCK_METHOD3(AddCreateRegKeyWorkItem,
+               WorkItem*(HKEY, const std::wstring&, REGSAM));
+  MOCK_METHOD3(AddDeleteRegKeyWorkItem,
+               WorkItem*(HKEY, const std::wstring&, REGSAM));
+  MOCK_METHOD4(
+      AddDeleteRegValueWorkItem,
+      WorkItem*(HKEY, const std::wstring&, REGSAM, const std::wstring&));
+  MOCK_METHOD2(AddDeleteTreeWorkItem,
+               WorkItem*(const base::FilePath&, const base::FilePath&));
+  MOCK_METHOD4(AddMoveTreeWorkItem,
+               WorkItem*(const base::FilePath&,
+                         const base::FilePath&,
+                         const base::FilePath&,
+                         MoveTreeOption));
+  // Workaround for gmock problems with disambiguating between string pointers
+  // and DWORD.
+  WorkItem* AddSetRegValueWorkItem(HKEY a1,
+                                   const std::wstring& a2,
+                                   REGSAM a3,
+                                   const std::wstring& a4,
+                                   const std::wstring& a5,
+                                   bool a6) override {
+    return AddSetRegStringValueWorkItem(a1, a2, a3, a4, a5, a6);
+  }
+
+  MOCK_METHOD6(AddSetRegStringValueWorkItem,
+               WorkItem*(HKEY,
+                         const std::wstring&,
+                         REGSAM,
+                         const std::wstring&,
+                         const std::wstring&,
+                         bool));
+  MOCK_METHOD6(AddSetRegDwordValueWorkItem,
+               WorkItem*(HKEY,
+                         const std::wstring&,
+                         REGSAM,
+                         const std::wstring&,
+                         DWORD,
+                         bool));
+};
+
+#if BUILDFLAG(ENABLE_BRAVE_VPN)
```

Comment:
why is this here when the entire file is excluded by the enable_brave_vpn flag?

Reply:
Removed in https://github.com/brave/brave-core/pull/20754/commits/a533769f71fd425aae4a05b94ce4245a1daada90